<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_Dialog</class>
 <widget class="QDialog" name="Add_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-size:13px;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Food name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="food_plainTextEdit">
         <property name="maximumSize">
          <size>
           <width>651</width>
           <height>31</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="recipe_label">
         <property name="minimumSize">
          <size>
           <width>619</width>
           <height>13</height>
          </size>
         </property>
         <property name="text">
          <string>Recipe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="recipe_textEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Ingredients</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="item_label">
           <property name="minimumSize">
            <size>
             <width>29</width>
             <height>29</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>29</width>
             <height>29</height>
            </size>
           </property>
           <property name="text">
            <string> Item:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="ingr_plainTextEdit">
           <property name="minimumSize">
            <size>
             <width>167</width>
             <height>29</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>167</width>
             <height>29</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="quant_label_2">
           <property name="minimumSize">
            <size>
             <width>46</width>
             <height>29</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>46</width>
             <height>29</height>
            </size>
           </property>
           <property name="text">
            <string>Quantity:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="quant_plainTextEdit">
           <property name="minimumSize">
            <size>
             <width>167</width>
             <height>29</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>167</width>
             <height>29</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="unit_label">
           <property name="minimumSize">
            <size>
             <width>23</width>
             <height>29</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>23</width>
             <height>29</height>
            </size>
           </property>
           <property name="text">
            <string>Unit:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="unit_plainTextEdit">
           <property name="minimumSize">
            <size>
             <width>167</width>
             <height>29</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>167</width>
             <height>29</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>458</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="add_ing_pushButton">
           <property name="text">
            <string>Add this ingredient</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPushButton" name="save_pushButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>418</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
